### PR TITLE
Docs for UXP v8.0

### PR DESCRIPTION
**Pull request by padmkris123**
Docs for UXP v8.0

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have signed the [Adobe Open Source CLA](https://opensource.adobe.com/cla.html).
- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: src/pages/uxp-api/reference-js/Global Members/Data Transfers/Clipboard.md [hunk 1]
```diff
@@ -75,11 +75,11 @@ Write text to clipboard. This can be used to implement cut and copy text functio
 
 | Param | Type | Description |
 | --- | --- | --- |
-| text | `object` | The text to set. Note, currently this will fail unless this is an object of the form {"text/plain": "text to set"}. This will be fixed in a future release. |
+| text | `string` \| `object` | text string to set or an object of the form {"text/plain": "text to set"}.                               Note that the object format will be deprecated and shouldn't be used. |
```
Comment (padmkris123) [inline src/pages/uxp-api/reference-js/Global Members/Data Transfers/Clipboard.md:78]: Any deprecated method should be called out separately in the changelog3P.md

File: src/pages/uxp-api/reference-js/Global Members/Data Transfers/fetch.md [hunk 2]
```diff
@@ -46,7 +43,7 @@ From UXP v7.4.0 onwards `permissions.network.domains` does not support wildCards
 | [init] | `Object` | `(Optional)` Custom settings for a HTTP request. |
 | [init.method] | `string` | HTTP request method. The default value is "GET". |
 | [init.headers] | `Headers` | HTTP request headers to add. |
-| [init.body] | `string` \| `ArrayBuffer` \| `TypedArray` \| `Blob` \| `FormData` \| `ReadableStream` \| `URLSearchParams` | Body to add to HTTP request. |
-| [init.credentials] | `string` | Indicates whether to send cookies. The default value is "include". Possible values and functions are as follows: <ul style="list-style: none;">  <li>"omit" : cookies are NOT sent.</li><li>"include" : cookies are sent.</li> </ul> |
+| [init.body] | `string` \| `ArrayBuffer` \| `TypedArray` \| `Blob` \| `FormData` \| `URLSearchParams` | Body to add to HTTP request. |
```
Comment (padmkris123) [inline src/pages/uxp-api/reference-js/Global Members/Data Transfers/fetch.md:46]: If we dont support `ReadableStream`, we should call that out as a breaking change.

File: src/pages/uxp-api/reference-js/Global Members/HTML DOM/Element.md [hunk 3]
```diff
@@ -317,6 +317,48 @@ Returns the open shadow root that is hosted by the element, or null if no open s
...
+
+| Param | Type |
+| --- | --- |
+| ...nodes | `Array<Node>` | 
+
+
+
+<a name="element-replacechildren" id="element-replacechildren"></a>
+
+## replaceChildren(...nodes)
```
Comment (padmkris123) [inline src/pages/uxp-api/reference-js/Global Members/HTML DOM/Element.md:350]: Is it replaceChildren or replaceWith?

File: src/pages/uxp-api/reference-js/Global Members/HTML Elements/HTMLVideoElement.md [hunk 4]
```diff
@@ -524,7 +548,22 @@ Resets the media to its initial state and begins the process of selecting a medi
 and loading the media in preparation for playback.
 The amount of media data that is prefetched is determined by the value of 'preload' attribute.
 
-**Emits**: `event:uxpvideoload`  
+**Emits**: `event:loadedmetadata`, `event:loadeddata`, `event:uxpvideoload - Deprecated: Use loadeddata instead`
```
Comment (padmkris123) [inline src/pages/uxp-api/reference-js/Global Members/HTML Elements/HTMLVideoElement.md:551]: `event:uxpvideoload` should be called out as deprecated in changelog3p.md

File: src/pages/uxp-api/reference-js/Global Members/HTML DOM/Element.md (hunk 3 above)
Comment (bhatiaPankaj) [reply src/pages/uxp-api/reference-js/Global Members/HTML DOM/Element.md:350]: It is `replaceChildren`
[ref](https://git.corp.adobe.com/torq/torq-native/blob/d828161a8816d0bbd66fd149d497f7122d7f48e5/modules/bindings/domjs/src/js/dom/nodes/Element.js#L338)

File: src/pages/uxp-api/reference-js/Global Members/Data Transfers/fetch.md (hunk 2 above)
Comment (bhatiaPankaj) [reply src/pages/uxp-api/reference-js/Global Members/Data Transfers/fetch.md:46]: This is just a documentation correction.

File: src/pages/uxp-api/reference-js/Global Members/Data Transfers/Clipboard.md (hunk 1 above)
Comment (bhatiaPankaj) [reply src/pages/uxp-api/reference-js/Global Members/Data Transfers/Clipboard.md:78]: Added a section `deprecations` to highlight this

File: src/pages/uxp-api/reference-js/Global Members/HTML DOM/Element.md (hunk 3 above)
Comment (padmkris123) [reply src/pages/uxp-api/reference-js/Global Members/HTML DOM/Element.md:350]: The Changelog3P says replaceWith.

**Pull request merged**